velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -831,6 +831,105 @@ DEBUG_ONLY_TEST_F(SharedArbitrationTest, reclaimFromAggregation) {
   }
 }
 
+DEBUG_ONLY_TEST_F(SharedArbitrationTest, reclaimFromDistinctAggregation) {
```

Comment:
Would it make sense to move this test to AggregationTest?

Reply:
Yes, will do after making memory arbitration the default in unit test. Will consolidate.